### ISSUE TITLE
Visible binding does not add aria-hidden="true" when hidden

### ISSUE THREAD

**Issue by CalvinRodo** [knockout/knockout#1322]
I work on a Canadian Government website and we are integrating knockout.js into our app. 
Because we have to have all of our sites 100% accessible I need to have all hidden nodes have aria-hidden="true" added. It should be removed when it's visible.

I've got a pull request coming that solves this, if it get's merged I'll start adding more accessibility features to the framework.
